samples/bluetooth/mesh/sensor_server/src/model_handler.c

```diff
@@ -65,21 +79,100 @@ static int chip_temp_get(struct bt_mesh_sensor_srv *srv,
 
 	return err;
 }
+/* Tolerance is based on nRF52832's range and accuracy (5/125 = 4%). */
+static const struct bt_mesh_sensor_descriptor chip_temp_descriptor = {
```

Comment:
The descriptor has been added, but doesn't really do anything on its own. I was able to print the info using the shell commands (descriptor get). Could for instance make one of the buttons trigger sending the message instead?

Reply:
Ye, sure you could try that